src/sentry/tasks/deliver_from_outbox.py

```diff
@@ -88,19 +91,20 @@ def schedule_batch(
             # Notably, when l and h are close, this will result in creating tasks that are processing future ids --
             # that's totally fine.
             for i in range(concurrency):
-                drain_task.delay(
-                    outbox_name=outbox_name,
-                    outbox_identifier_low=lo + i * batch_size,
-                    outbox_identifier_hi=lo + (i + 1) * batch_size,
-                )
+                if hasattr(drain_task, "delay"):
```

Comment:
this points at the types here being incorrect -- before it was unconditionally calling this meaning the parameter is wrong and that should get fixed instead